keyboards/handwired/ms_sculpt_mobile/ms_sculpt_mobile.h

```diff
@@ -1,43 +1,90 @@
+/*
+ * Copyright 2021 milestogo
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
 #pragma once
 
+
+#pragma once 
 #include "quantum.h"
+ 
+#define XXX KC_NO
+#define NNN KC_NO
 
 #define XXX KC_NO
 
-#define LAYOUT( \
-    k7G, k6E, k6F, k6G, k5E, k5F, k5G, k70, k71, k72, k73, k74, k75, k1E, k1A, k1B, \
-    k50, k5A, k5B, k5C, k5D, k57, k58, k20, k21, k22, k23, k51, k52, k59, k24, \
-    k6H, k63, k64, k65, k78, k79, k40, k41, k42, k43, k30, k31, k32, k33, \
-    k09, k60, k61, k62, k77, k10, k11, k53, k54, k55, k56, k12, k7F, k26, \
-    k2F, k7A, k7B, k7C, k7E, k00, k01, k02, k03, k04, k05, k2B, k66, k1F, \
-    k1G, k4D, k3E, k6D, k3A, k0H, k1C, k67, k68, k69 \
-) { \
-    { k00, k01, k02, k03, k04, k05, XXX, XXX, XXX, k09, XXX, XXX, XXX, XXX, XXX, XXX, XXX, k0H }, \
-    { k10, k11, k12, XXX, XXX, XXX, XXX, XXX, XXX, XXX, k1A, k1B, k1C, XXX, k1E, k1F, k1G, XXX }, \
-    { k20, k21, k22, k23, k24, XXX, k26, XXX, XXX, XXX, XXX, k2B, XXX, XXX, XXX, k2F, XXX, XXX }, \
-    { k30, k31, k32, k33, XXX, XXX, XXX, XXX, XXX, XXX, k3A, XXX, XXX, XXX, k3E, XXX, XXX, XXX }, \
-    { k40, k41, k42, k43, XXX, XXX, XXX, XXX, XXX, XXX, XXX, XXX, XXX, k4D, XXX, XXX, XXX, XXX }, \
-    { k50, k51, k52, k53, k54, k55, k56, k57, k58, k59, k5A, k5B, k5C, k5D, k5E, k5F, k5G, XXX }, \
-    { k60, k61, k62, k63, k64, k65, k66, k67, k68, k69, XXX, XXX, XXX, k6D, k6E, k6F, k6G, k6H }, \
-    { k70, k71, k72, k73, k74, k75, XXX, k77, k78, k79, k7A, k7B, k7C, XXX, k7E, k7F, k7G, XXX } \
+//US layout, with vol-, vol+, mute in upper right, white Fn keys
+#define LAYOUT_mobile_XUA( \
```

Comment:
Layout macro named should be fully lowercase except for the `LAYOUT` prefix.

* [Keyboard Guidelines - `<keyboard_name.h>` - QMK Firmware Documentation](https://docs.qmk.fm/#/hardware_keyboard_guidelines?id=keyboard-folder-structure)

```suggestion
#define LAYOUT_mobile_xua( \
```